include/splinterdb/splinterdb.h

```diff
@@ -50,6 +50,23 @@ typedef struct {
    bool        cache_use_stats;
    const char *cache_logfile;
 
+   // task system
+   // Background threads configuration: Both have to be non-zero in order for
```

Comment:
@rtjohnso -- heads-up: Seems like you've removed this restriction that both have to be non-zero (or zero).

This comment block could come out now, otherwise, it will be inaccurate.